cmd/ltctl/report.go

```diff
@@ -90,6 +90,11 @@ func RunGenerateReportCmdF(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("could not parse output: %w", err)
 		}
+
+		if !output.HasMetrics() {
+			return fmt.Errorf("no active deployment found, use the `--prometheus-url` flag.\nIf you have a local or manually deployed Prometheus server running")
```

Comment:
No need to break the line. It's a single error message. Let the user's console break the line for us :)

Reply:
Makes sense :)) I've just pushed the removal